test/mainnet/compounder/curve/testETHfrxETH.sol

```diff
@@ -73,6 +73,12 @@ contract testETHfrxETH is Test, AddRoutes, CurveCompounderBaseTest {
         _testMint(frxETH, _amount);
     }
 
+    function testDepositCap(uint256 _amount) public {
+        vm.assume(_amount > 0.01 ether && _amount < 5 ether);
```

Comment:
Why do we assume that ETH range? Or what is being assumed here exactly?

Reply:
My expectation would be that there are specific values we end up with. Even if they differ between test cases. Or is it that the values differ across test executions? Because that should not be the case under any circumstances. The only cases in which I would do anything like this is for accounting for rounding errors when working with floating point numbers. But that is not the case here. I think we should be as specific as we can.

Reply:
So what usually happens if we don't gate the fuzz tests values is that when swapping ETH to some asset we need for testing (an operation that happens in every test that needs assets that are not ETH/WETH, and by 3 or more demo users), it swaps an amount the creates too much slippage in the pool, it is not fixed by MEV obviously, and so other demo users are swapping with very high slippage, which messes up the rest of the test flow.

We could gate the value from 0.00001 ETH to 100 ETH for example, but then on some flows it will work (as there's enough liquidity), and on others it wont. Basically as we allow bigger numbers to be used the chance the test flow gets messed up by slippage increases. 

0.01 Ether to 5 Ether seems to by a good default amount to use.